cypress/e2e/app/simple-e2etest.cy.js

```diff
@@ -70,44 +72,50 @@ describe("End to end test using a simple UI path through the app", () => {
                 // B. Categorize "participant_id" as "Subject ID"
                 cy.categorizeColumn("Subject ID", p_dataset["category_columns"]["Subject ID"][0]);
 
-                // C. Assert nav and next button are enabled
-                cy.assertNextPageAccess("annotation", true);
+                // C. Assert nav and next button are not yet enabled
+                cy.assertNextPageAccess("annotation", false);
 
                 // D. Categorize "age" as "Age"
                 cy.categorizeColumn("Age", p_dataset["category_columns"]["Age"][0]);
 
+                cy.assertNextPageAccess("annotation", true);
+
                 // E. Click the next page button to proceed to the categorization page
                 cy.nextPageByButton();
 
-                // 3. Go through annotation page, saving default age annotation
+                cy.commitToVuexStore("setCurrentPage", "annotation");
 
-                // A. Assert that next page nav and button are disabled for download page
-                cy.assertNextPageAccess("download", false);
+                if ( cy.datasetMeetsTestCriteria("annotation", p_dataset, testCriteria)) {
 
-                // B. Click on the 'Age' tab
-                cy.get("[data-cy='annotation-category-tabs'] ul")
-                    .contains("li", "Age")
-                    .click();
+                    // 3. Go through annotation page, saving default age annotation
 
-                // C. Click on the 'Save Annotation' button
-                cy.get("button")
-                    .contains("Save Annotation")
-                    .click();
+                    // A. Assert that next page nav and button are disabled for download page
+                    cy.assertNextPageAccess("download", false);
 
-                // D. Assert that next page nav and button are enabled for download page
-                cy.assertNextPageAccess("download", true);
+                    // B. Click on the 'Age' tab
+                    cy.get("[data-cy='annotation-category-tabs'] ul")
+                        .contains("li", "Age")
+                        .click();
 
-                // E. Click the next page button to proceed to the download page
-                cy.nextPageByButton();
+                    // B. Select the 'float' transformation heuristic
+                    // :data-cy="'selectTransform_' + columnName"
```

Comment:
nice, this line is helpful

Reply:
you might even spell out the full sentence: "and `columnName` is `age`" or something like that to make it even clearer